claim/claim.go

```diff
@@ -19,10 +19,12 @@ const DefaultSchemaVersion = schemaversion.SchemaVersion("v1.0.0-WD")
 
 // Status constants define the CNAB status fields on a Result.
 const (
-	StatusSuccess = "success"
-	StatusFailure = "failure"
-	StatusPending = "pending"
-	StatusUnknown = "unknown"
+	StatusSuccess  = "succeeded"
+	StatusCanceled = "canceled"
+	StatusRunning  = "running"
+	StatusFailure  = "failed"
```

Comment:
This might be controversial - but do we get any value from also updating the variable name as well - e.g. `StatusFailed`?

Reply:
I didn't suggest it because it would require even more code changes for consumers but with very little reward. 😀 But now is the time to make breaking changes so if we do want it to match exactly, I can make that change?

Reply:
I vote for updating the names but we can also add compatibility constants, documented as depecated:
```go
const {
	StatusSucceeded  = "succeeded"
	StatusCanceled = "canceled"
	StatusFailed  = "failed"
	StatusRunning  = "running"
	StatusPending  = "pending"
	StatusUnknown  = "unknown"
        // DEPRECATED
	StatusSuccess = StatusSucceeded
	StatusFailure = StatusFailed
}
```